eemeter/modeling/models/seasonal.py

```diff
@@ -195,7 +200,32 @@ def fit(self, input_data):
         }
         return output
 
-    def predict(self, demand_fixture_data, params=None):
+    def _bootstrap_empirical_errors(self):
+        ''' Calculate empirical bootstrap error function '''
+        self.N_bootstrap = 100
+        if len(self.X) < 200:
+            return lambda N: self.rmse * (N**0.8)
```

Comment:
Looks like there are three different ways to calculate errors - this one (fallback if bootstrapping will fail), the model.upper and model.lower (used if summed=False), and the bootstrapping method. Is that right?